devel/meson/Makefile

```diff
@@ -0,0 +1,56 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=meson
+PKG_VERSION:=0.52.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/mesonbuild/meson/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=0f426ed1362c38bcc5b9027ec6aec3445d6db88e8d7249ed992e9af88a42d0e0
+
+PKG_MAINTAINER:=Andre Heider <a.heider@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+HOST_BUILD_DEPENDS:=ninja/host
```

Comment:
not needed. ninja already depends on its host build.

Reply:
Meson requires ninja at runtime, and ports built with meson use `PKG_BUILD_DEPENDS:=meson/host` which then drags ninja/host in with this. How else do I express this (The 'BUILD' part of the name is confusing, but I didn't find anything a la HOSTPKG_DEPENDS)?

Reply:
Right, but ninja is listed as a dependency. ninja/host is picked up from there.

Reply:
The package does, but that didn't make the host build depend on it IIRC. I can check again though

Reply:
If I remove that line, the host meson does not depend on host ninja anymore. So it breaks compiling packages with meson